var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -90,9 +90,13 @@ def configure_args(self):
             '-des',
             '-Dprefix={0}'.format(prefix),
             '-Dlocincpth=' + self.spec['gdbm'].prefix.include,
-            '-Dloclibpth=' + self.spec['gdbm'].prefix.lib,
```

Comment:
Please leave the comma at the end of the line, it's a convenience that minimizes diffs if anyone ever adds another line to the list and helps avoid the "oops I forgot to add a comma to the previous line" error.